cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -17,6 +17,13 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 // CNV-2928/CNV-4166 OVN-Kubernetes network provider
 
+// CNV-3535 Dedicated resources
+* A new option to view and configure dedicated resources for virtual machines is
+available in the web console *Details* view. When you enable dedicated
```

Comment:
This will fall under a === Web console heading, and it should be 'Virtual Machine Details', so I suggest changing the end of the first sentence to:
"...available in the *Virtual Machine Details* page."

Reply:
Thanks. I moved this to the last entry (after Networking and before Notable Technical Changes) and also corrected the comment CNV number (I had 3535, which is the Jira story for the actual feature doc):

=== Web console
// CNV-4076 UI view/config dedicated resources
* A new option to view and configure dedicated resources for virtual machines is
available in the web console *Virtual Machine Details* view. When you enable dedicated
resources, you ensure that your virtual machine's workload runs on CPUs that
are not used by other processes, improving your virtual machine's performance
and accuracy of latency predictions.